include/deal.II/trilinos/nox.h

```diff
@@ -61,6 +75,11 @@ namespace TrilinosWrappers
    * // the final solution
    * solver.solve(solution);
    * @endcode
+   *
+   * The functions used in NOX are nearly identical to the functions in KINSOL
+   * with a few exceptions (we do not call reinit() in NOX). So checl the KINSOL
```

Comment:
```suggestion
   * with a few exceptions (we do not call reinit() in NOX). So check the KINSOL
```

What do you mean by "we do not call reinit() in NOX"?

Reply:
With step-77, KINSOL requires calling 4 functions, reinit(), residual(), compute_jacobian(), and solve(). NOX does not need reinit().

A better sentence might be "KINSOL requires a reinit() function where NOX does not", does that make more sense?

Reply:
I see. I just realized the NOX does not even have the ``residual`` function slot.

Reply:
@tjhei Actually, NOX has a `residual`  function slot as well ;)

Reply:
Sorry, I meant ``reinit``.